client/components/seo/seo-site-search-preview.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { PropTypes } from 'react';
+import toString from 'lodash/toString';
+
+const reactifyNewlines = message =>
+	message
+		.split( '\n' )
+		.map( ( snip, index ) => [ <span key={ `s${ index }` }>{ snip }</span>, <br key={ `br${ index }` } /> ] )
+		.reduce( ( final, next ) => final.concat( next ), [] )
+		.slice( 0, -1 );
+
+export const SeoSiteSearchPreview = React.createClass( {
```

Comment:
Since you're not wrapping the default export, could you just set the `default` export here instead of bottom of file, or is this for consistency with other components where the base component may or may not be wrapped?